src/BedrockStreaming.php

```diff
@@ -8,16 +8,19 @@
 
 final class BedrockStreaming extends Config
 {
-    public function __construct()
+    /** @var array $customRules */
+    private $customRules;
+    public function __construct(array $customRules = [])
```

Comment:
I thought that the whole point of `BedrockStreaming/php-cs-fixer-config` was to share the exact same rules between all PHP teams at Bedrock... Isn't it weird to allow custom rules, then? 🤔

Reply:
You're right. We do wish to keep sharing the exact same cs-fixer rules between the company PHP teams.
However, this has some limitations. In case of specific needs per team / project, then we happen to decide extra rules. For now we have no solution to apply them properly and automatically.
This features quarantees that those extra rules do not override the ones defined at the Bedrock PHP teams level.